### PR TITLE
Bsp update

### PR DESCRIPTION
**Pull request by vicgal**

### CODE REVIEW

File: layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend [hunk 1]
```diff
@@ -19,6 +19,7 @@ SRC_URI_append_etcher-pro = " \
 	file://0001-Add-support-for-KSZ9893R.patch \
 	file://0001-Enable-KSZ9893R-switch-in-Etcher-Pro-dtb.patch \
 	file://0007-HDMI-Set-no-EDID.patch \
+	file://0008-Add_ep_config.patch \
```
Comment (floion) [inline layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend]: You should mention in the git log where this config is taken from.
And if you want to add this defconfig, you can just add it as a file in the yocto layer and configure the kernel to use it. As this patch adds it now, how is the build system instructed to use this defconfig?
Also, as a NIT there is a typo in the git log: `requried`

Comment (vicgal) [reply layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend]: I think it's due to the BSP update. The defconfig is linked to the machine name. When I triggered the etcher-pro build, it failed, asking for etcher-pro_defconfig

Comment (vicgal) [reply layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend]: done

File: layers/meta-balena-compulab/recipes-kernel/linux/linux-imx/0001-Revert-MLK-17537-8-drm-mxsfb-Add-support-for-mode_va.patch [hunk 2]
```diff
@@ -0,0 +1,366 @@
+From 649d1d63c8e9e9e6c499e14429d5d90341e85b3f Mon Sep 17 00:00:00 2001
+From: Robert Chiras <robert.chiras@nxp.com>
+Date: Tue, 28 May 2019 17:27:48 +0300
+Subject: [PATCH 1/7] Revert "MLK-17537-8: drm/mxsfb: Add support for
+ mode_valid"
+
+This reverts commit 01ba61aee0e2487e14c11c24a0eb3787fd3c990c.
```
Comment (floion) [inline layers/meta-balena-compulab/recipes-kernel/linux/linux-imx/0001-Revert-MLK-17537-8-drm-mxsfb-Add-support-for-mode_va.patch:7]: These patches lack `Signed-off-by` and `Upstream-status`

**Pull request merged**